arduino/scout/Frequency.h

```diff
@@ -9,11 +9,15 @@ class Frequency {
   Frequency(float glide, int cyclesPerGlideMax);
   void update(float target, float glide);
   float get();
+  uint16_t getTicks();
+  uint16_t getPeriod();
   void reset();
   void print();
 
 private:
   float _frequency = 0;
+  uint16_t _ticks = 0;
```

Comment:
These ticks are no longer used (in favor of using the period). If you don't mind the dithering at higher pitches, I could get rid of them. Though I guess it doesn't hurt to leave them in.